### PR TITLE
add otakufile

### PR DESCRIPTION
**Pull request by arifr007**

### CODE REVIEW

File: src/id/otakufile/src/eu/kanade/tachiyomi/extension/id/otakufile/OtakuFile.kt [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+
+    override fun popularMangaRequest(page: Int): Request {
+        return GET("$baseUrl/komik-populer", headers)
+    }
+
+    override fun latestUpdatesRequest(page: Int): Request {
+        return GET("$baseUrl/", headers)
+    }
+
+    override fun searchMangaRequest(page: Int, query: String, filters: FilterList): Request = latestUpdatesRequest(page)
```
Comment (arkon) [inline src/id/otakufile/src/eu/kanade/tachiyomi/extension/id/otakufile/OtakuFile.kt]: You can probably make a request to the endpoint that the site uses for that dropdown via AJAX.

Comment (arifr007) [reply src/id/otakufile/src/eu/kanade/tachiyomi/extension/id/otakufile/OtakuFile.kt]: @arkon ill add search feature in next update

Comment (arifr007) [reply src/id/otakufile/src/eu/kanade/tachiyomi/extension/id/otakufile/OtakuFile.kt]: > You can probably make a request to the endpoint that the site uses for that dropdown via AJAX.

can you give me example request using ajax? i haven't use ajax in kotlin before ._.

Comment (arkon) [reply src/id/otakufile/src/eu/kanade/tachiyomi/extension/id/otakufile/OtakuFile.kt]: I mean the website hits an endpoint via AJAX. It'd just be another regular request in the extension.

**Pull request merged**